test/rtp_decoder.c

```diff
@@ -207,20 +267,35 @@ int main(int argc, char *argv[])
             scs = *i_scsp;
             input_key = malloc(scs.key_size);
             sec_servs |= sec_serv_conf | sec_serv_auth;
+            gcm_on = scs.gcm_on;
             break;
         default:
             usage(argv[0]);
         }
     }
 
+    if (scs.tag_size == 0) {
+        if (gcm_on) {
+            scs.tag_size = 16;
+        } else {
+            scs.tag_size = 10;
+        }
+    }
+
     if (gcm_on && scs.tag_size != 8 && scs.tag_size != 16) {
         fprintf(stderr, "error: GCM tag size must be 8 or 16 (%d)\n",
```

Comment:
Size 8 is not allowed for GCM per the spec.  Should we allow it here?  That would mean we remove srtp_crypto_policy_set_aes_gcm_128_8_auth as an option.

Reply:
As discussed I don't see anything that uses a tag size of 8 for GCM w/ SRTP or SDES. The only valid value is 16.

https://www.iana.org/assignments/sdp-security-descriptions/sdp-security-descriptions.xhtml

https://tools.ietf.org/html/rfc7714
> 1.  Introduction
>       AEAD_AES_128_GCM      AES-128 with a **16-octet authentication tag**
>       AEAD_AES_256_GCM      AES-256 with a **16-octet authentication tag**

Reply:
Since the -s option with AEAD_AES_128_GCM or AEAD_AES_256_GCM will set the tag length to 16 and the support for tag of 8 has been there for a while then I don't think I would want to change that in this commit.  Will instead open a new issue to potentially remove srtp_crypto_policy_set_aes_gcm_128_8_auth and friends from srtp.h, and if that happens then rtp_decoder will also be changed to remove support for 8 byte tag.

Reply:
#441 created to track this.